lnrpc/rpc.proto

```diff
@@ -743,90 +743,97 @@ message HTLC {
     uint32 expiration_height = 4 [json_name = "expiration_height"];
 }
 
-message ActiveChannel {
+message Channel {
     /// Whether this channel is active or not
     bool active = 1 [json_name = "active"];
 
+    /// Whether this channel is advertised to the network or not
+    bool private = 2 [json_name = "private"];
```

Comment:
in order to maintain backwards compatibility for proto files, sequence numbers for existing field should never change. this field should be added to the end of the definition, and the indexes of prior fields should have their indexes reverted

Reply:
Good point. I believe since we don’t actually store the serialized data of the invoice proto anywhere, it _should_ be fine. I could be mistaken though. I’ll apply your suggestion for consistency regardless.

Reply:
You are right, for our use case it's probably okay. But that doesn't preclude others from storing/continuing to use the proto definitions ;)

Reply:
Definitely! Will fix.